benches/benches/transaction_throughput.rs

```diff
@@ -179,16 +181,22 @@ fn signed_transfers(c: &mut Criterion) {
                 Default::default(),
                 Default::default(),
             )
-            .add_unsigned_coin_input(
-                SecretKey::random(rng),
+            .add_input(Input::coin_predicate(
                 rng.gen(),
+                owner,
                 1000,
                 Default::default(),
                 Default::default(),
-            )
+                Default::default(),
+                predicate.clone(),
+                vec![],
+            ))
             .add_output(Output::coin(rng.gen(), 50, AssetId::default()))
             .add_output(Output::change(rng.gen(), 0, AssetId::default()))
-            .finalize()
+            .finalize();
+        tx.estimate_predicates(&checked_parameters(), MemoryInstance::new())
+            .expect("Predicate check failed");
+        tx
     };
     bench_txs("signed transfers", c, generator);
```

Comment:
Should we rename this to "predicate transfer" now when we use a predicate instead of a signed coin input?